src/interactors/LearningObjectInteractor.ts

```diff
@@ -32,17 +32,17 @@ export class LearningObjectInteractor {
     library: LibraryCommunicator;
     username: string;
     userToken: UserToken;
-    accessUnpublished?: boolean;
+    accessUnreleased?: boolean;
     loadChildren?: boolean;
     query?: LearningObjectQuery;
   }): Promise<LearningObject[]> {
     try {
-      // Set accessUnpublished
-      let accessUnpublished = params.accessUnpublished;
+      // Set accessUnreleased
+      let accessUnreleased = params.accessUnreleased;
 
-      // If accessUnpublished is unset, set equal to the result of the hasOwnership function
-      if (accessUnpublished === undefined || accessUnpublished === null) {
-        accessUnpublished = await this.hasOwnership({
+      // If accessUnreleased is unset, set equal to the result of the hasOwnership function
+      if (accessUnreleased === undefined || accessUnreleased === null) {
```

Comment:
This could be simplified by using the double equals expression below, so that type coercion is not used and both null/undefined will register as being a bottom value:

```suggestion
      if (accessUnreleased == null) {
```